.github/workflows/ci.yml

```
@@ -0,0 +1,315 @@
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ['3.9', '3.10', '3.11', '3.12']
+        include:
+          # Test with development versions on latest OS
+          - os: ubuntu-latest
+            python-version: '3.13-dev'
+            experimental: true
+        exclude:
```

actionlint (1.7.8)

expecting a single ${{...}} expression or array of matrix combination, but found plain text node

(syntax-check)